traffic_ops/app/lib/API/Tenant.pm

```diff
@@ -117,14 +123,25 @@ sub update {
 	my $is_active = $params->{active};
 	
 	if ( !$params->{active} && $self->isRootTenant($id)) {
```

Comment:
can we just make root tenant uneditable? for example:

if $self->isRootTenant($id) return $self->alert("Cannot edit root tenant.");